index.js

```diff
@@ -40,6 +40,12 @@ export default function stringWidth(string, options = {}) {
 			continue;
 		}
 
+		// For grapheme contains more than one code points
```

Comment:
I don't think this is correct. Graphemes can have two code points and still take up a `1` width.

Reply:
Yea, I can see that from test, but not sure what's correct.

Reply:
`[...grapheme].some(codePoint => eastAsianWidth(codePoint ) > 2)`?

Reply:
I'm honestly not sure. The East Asian Width spec doesn't provide any guidance for that. From what I have seen from other implementations, all others use the first code point too.